### PR TITLE
Identify generated files so GitHub can default to collapsing their diffs

### PR DESCRIPTION
**Pull request by baumanj**
See https://github.com/github/linguist for more

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,2 @@
+components/sup-protocol/src/generated/*.rs linguist-generated
```
Comment (raskchanky) [inline .gitattributes]: The example at https://github.com/github/linguist#generated-code has `linguist-generated=true`.  Is the `=true` required?

Comment (baumanj) [reply .gitattributes]: Probably not; I took my inspiration from https://robots.thoughtbot.com/github-diff-supression, but I guess we'll see.

**Comment by christophermaier** [comment]
tenor-173784934

**Pull request merged**